### DISCUSSION TITLE
Live: filling with data on the chart.

### DISCUSSION THREAD

**Discussion by ihard** [grafana/grafana#40585] [Dashboards, Panels & Visualizations]
Live metrics can now use Redis to synchronize data between instances, but it seems that it can also be a small cache with data for a few minutes to be able to fill the graph the first time it goes to it (so as not to wait until it is filled with values). 
This would greatly improve the usability.
Perhaps there is already a request for this, I would be grateful for the link, thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
